Makefile

```diff
@@ -470,7 +470,7 @@ test-junit: $(SETUP_ENVTEST) $(GOTESTSUM) $(GOVC) ## Run unit tests
 
 .PHONY: test-cover
 test-cover: ## Run unit tests and generate a coverage report
-	$(MAKE) test TEST_ARGS="$(TEST_ARGS) -coverprofile=coverage.out"
+	$(MAKE) test TEST_ARGS="$(TEST_ARGS) -coverpkg=./apis/...,./controllers/...,./pkg/...,./internal/... -coverprofile=coverage.out"
```

Comment:
Would all packages be covered by `./...` instead of naming each package individually.

Reply:
I think it should work

Reply:
I believe so. IMO we should include the entire codebase for our coverage baseline.

Reply:
Yup will include that ones as well. I think it's fine to include all code. We just shouldn't use the one number to judge anything by. But we would always have the problem. There are more packages where we could discuss if they require test coverage or not. I think in the end we have to look package by package where we should add more tests and we can just skip packaging there.